### PR TITLE
Mondo master: Antiforgery CORS HttpSysServer ResponseCaching Routing and Security

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

File: build/buildorder.props [hunk 1]
```diff
@@ -8,18 +8,18 @@
 
   <ItemGroup>
     <RepositoryBuildOrder Include="Razor" Order="6" />
-    <RepositoryBuildOrder Include="HttpSysServer" Order="8" />
+    <RepositoryBuildOrder Include="HttpSysServer" Order="8" RootPath="$(RepositoryRoot)src\HttpSysServer\" />
     <RepositoryBuildOrder Include="BasicMiddleware" Order="9" />
-    <RepositoryBuildOrder Include="Antiforgery" Order="10" />
+    <RepositoryBuildOrder Include="Antiforgery" Order="9" RootPath="$(RepositoryRoot)src\Antiforgery\" />
```
Comment (natemcmaster) [inline build/buildorder.props]:
```suggestion
    <RepositoryBuildOrder Include="Antiforgery" Order="10" RootPath="$(RepositoryRoot)src\Antiforgery\" />
```

Just to be safe, avoid build order changes.

**Review by natemcmaster — APPROVED** [review]
One minor change, then :shipit: (assuming CI builds still pass)

**Comment by ryanbrandenburg** [comment]
Failures are Kestrels fault (@muratg). Merging.

**Review by muratg — COMMENTED** [review]
cc @halter73 @Tratcher

**Pull request merged**